### PR TITLE
[blog] Exploring NATS as a backend for k3s

### PR DESCRIPTION
**Pull request by bruth**

### CODE REVIEW

**Comment by bruth** [comment]
☝️ I am going try to add a short screen capture of k3s running with embedded NATS if I can get it to build/run on my Mac. Working on my Linux machine, but the screen capture software is poor.

**Comment by bruth** [comment]
@caleblloyd Added the image from your slide deck, Steve shared with me. Do you have a higher quality version?

File: content/blog/exploring-nats-as-a-backend-for-k3s.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ The `nats://` endpoint supports a `serverConfig` query param pointing to a local
 
 Building on the above example showcased in the RethinkConn talk, embedded NATS makes it possible to extend to the edge and maintain local state without remote connectivity to a NATS cluster. However, visibility and management of these edge (and non-edge) deployments can still be acheived.
 
-This segways into the second point, which is, acknowledging the combination of NATS embedded within k3s. A single binary gives you all the advantages of NATS combined with a workload scheduler!
+This segues into the second point, which is, acknowledging the combination of NATS embedded within k3s. A single binary gives you all the advantages of NATS combined with a workload scheduler!
```
Comment (bruth) [inline content/blog/exploring-nats-as-a-backend-for-k3s.md:81]: @gcolliso LOL.. nice catch.

**Comment by bruth** [comment]
Added a one-minute demo video and updated the image quality.

**Comment by bruth** [comment]
Unless @caleblloyd has an additional feedback, I think it is ready.

**Review by gcolliso — APPROVED** [review]
lgtm

**Review by caleblloyd — APPROVED** [review]
looks good!

**Pull request merged**